packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -56,7 +56,12 @@ export default function ImageSizeControl( {
 							value={ width ?? imageWidth ?? '' }
 							min={ 1 }
 							onChange={ ( value ) =>
-								onChange( { width: parseInt( value, 10 ) } )
+								onChange( {
+									width:
+										value === ''
```

Comment:
```suggestion
										!! value
```

Would it be worth changing the condition to `!! value` instead to also catch `0` width/height values? They probably don't make a lot of sense either.

Reply:
I don't think that will work cleanly since `value` is a string, so `'0'` will be truthy, but I can certainly add an additional check on the parsed value if you think that would be best. I don't feel strongly either way -- I agree that it doesn't make a lot of sense, but '0' is _technically_ a valid choice.

Reply:
Good point. That suggestion was a bit of an afterthought. I think it's fine to proceed with the existing check.